finance-info.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index d51267a..e1c5082 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -3,19 +3,17 @@ name = Finance
+ activity_version = 14
+ license = GPLv3+
+ host_version = 1
+-bundle_id = org.laptop.community.Finance
++release_date = 2018-04-12
++bundle_id = org.sugarlabs.Finance
+ icon = Finance-activity
+ exec = sugar-activity3 finance.Finance
+ summary = Want to wisely manage money? Learn financial planning basics and track your family's finance, make your own budget and learn how to make every penny count!
```

Comment:
One, one last comment, how about just `Learn financial planning basics` ? It says all a summary needs.

Reply:
Don't forget about this.

Reply:
I thought you mentioned about the description, so I updated the description instead :P